### PR TITLE
Editorial mdlint and vale updates in the References/CLI directory

### PR DESCRIPTION
**Pull request by grantnoble**
EOL spaces, line length, sentence case for headings, active voice, plus additional misc changes

### CODE REVIEW

**Comment by grantnoble** [comment]
Comments in some code blocks use "#" which markdown lint interprets as a level 1 heading and throws "MD022 Headers should be surrounded by blank lines".
Also a weird error I can't find: "./docs/Reference/CLI/CLI-Syntax.md:4: MD038 Spaces inside code span elements". I wonder if it's caused by the MD022 error?

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -1828,7 +1903,8 @@ BESU_RPC_WS_AUTHENTICATION_CREDENTIALS_FILE=/home/me/me_node/auth.toml
 rpc-ws-authentication-credentials-file="/home/me/me_node/auth.toml"
 ```
 
-[Credentials file](../../HowTo/Interact/APIs/Authentication.md#credentials-file) for JSON-RPC API [authentication](../../HowTo/Interact/APIs/Authentication.md).
+The [Credentials file](../../HowTo/Interact/APIs/Authentication.md#credentials-file) for JSON-RPC
```
Comment (NicolasMassart) [inline docs/Reference/CLI/CLI-Syntax.md]: Path for the file?

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -754,8 +780,8 @@ BESU_MIN_GAS_PRICE=1337
 min-gas-price=1337
 ```
 
-The minimum price that a transaction offers for it to be included in a mined block. To retrieve the minimum price in a running node, use [`eth_gasPrice`](../API-Methods.md#eth_gasprice).
-Default is 1000.
+The minimum price a transaction offers to include it in a mined block. To retrieve the minimum
+price in a running node, use [`eth_gasPrice`](../API-Methods.md#eth_gasprice). The default is 1000.
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/Reference/CLI/CLI-Syntax.md]: Done.

File: docs/Reference/CLI/CLI-Syntax.md [hunk 3]
```diff
@@ -836,10 +871,10 @@ BESU_NETWORK_ID=8675309
 network-id="8675309"
 ```
 
-[P2P network identifier](../../Concepts/NetworkID-And-ChainID.md).
+The [P2P network identifier](../../Concepts/NetworkID-And-ChainID.md).
 
-This option can be used to override the default network ID.
-The default value is the network chain ID defined in the genesis file.
+Use this option to override the default network ID. The default value is the network chain ID
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/Reference/CLI/CLI-Syntax.md]: Done.

File: docs/Reference/CLI/CLI-Syntax.md [hunk 4]
```diff
@@ -1289,7 +1341,7 @@ BESU_PRIVACY_TLS_KEYSTORE_PASSWORD_FILE=/home/me/me_node/password
 privacy-tls-keystore-password-file="/home/me/me_node/password"
 ```
 
-Path to a file that contains the password to decrypt the keystore.
+The file containing the password to decrypt the keystore.
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/Reference/CLI/CLI-Syntax.md]: Done.

File: docs/Reference/CLI/CLI-Syntax.md [hunk 5]
```diff
@@ -1753,7 +1825,7 @@ BESU_RPC_HTTP_TLS_KEYSTORE_PASSWORD_FILE=/home/me/me_node/password
 rpc-http-tls-keystore-password-file="/home/me/me_node/password"
 ```
 
-Path to a file that contains the password to decrypt the keystore.
+The file that contains the password to decrypt the keystore.
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/Reference/CLI/CLI-Syntax.md]: Done.

File: docs/Reference/CLI/CLI-Syntax.md [hunk 6]
```diff
@@ -1773,14 +1845,17 @@ BESU_RPC_HTTP_TLS_KNOWN_CLIENTS_FILE=/home/me/me_node/knownClients
 rpc-http-tls-known-clients-file="/home/me/me_node/knownClients"
 ```
 
-Used to [authenticate clients](../../HowTo/Configure/Configure-TLS.md#create-the-known-clients-file) that use self-signed certificates or non-public
-certificates.
+The file used to
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/Reference/CLI/CLI-Syntax.md]: Done.

File: docs/Reference/CLI/CLI-Syntax.md [hunk 7]
```diff
@@ -1309,8 +1361,8 @@ BESU_PRIVACY_TLS_KNOWN_ENCLAVE_FILE=/home/me/me_node/knownEnclave
 privacy-tls-known-enclave-file="/home/me/me_node/knownEnclave"
 ```
 
-Path to a file containing the hostnames, ports, and SHA256 certificate fingerprints
-of the [authorized privacy enclave](../../HowTo/Configure/Configure-TLS.md#create-the-known-servers-file).
+The file containing the hostnames, ports, and SHA256 certificate fingerprints of the
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/Reference/CLI/CLI-Syntax.md]: Done

File: docs/Reference/CLI/CLI-Syntax.md (hunk 1 above)
Comment (grantnoble) [reply docs/Reference/CLI/CLI-Syntax.md]: Done.

File: docs/Reference/CLI/CLI-Syntax.md [hunk 8]
```diff
@@ -1192,17 +1241,18 @@ BESU_PRIVACY_MULTI_TENANCY_ENABLED=false
...
 
 ```bash tab="Syntax"
 --privacy-precompiled-address=<privacyPrecompiledAddress>
 ```
 
-Address to which the [privacy pre-compiled contract](../../Concepts/Privacy/Private-Transaction-Processing.md) is mapped.
-The default is 126.     
-    
+The [privacy pre-compiled contract](../../Concepts/Privacy/Private-Transaction-Processing.md)
+address. The default is 126.
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/Reference/CLI/CLI-Syntax.md]: Done.

**Pull request merged**